docs/new-architecture-library-intro.md

```diff
@@ -130,6 +130,19 @@ In general, this means you can use primitive types (strings, numbers, booleans),
 
 > See Appendix [I. Flow Type to Native Type Mapping](./new-architecture-appendix#i-flow-type-to-native-type-mapping). (TypeScript to Native Type Mapping will be added soon.)
 
+### Codegen helper types
+
+You can use predefined types for your JavaScript spec, here is a list of them:
+
+- `Double`
+- `Float`
+- `Int32`
+- `WithDefault<Type, Value>` - Sets default value for type
+- `BubblingEventHandler<T>` - For bubbling events (eg: `onChange`).
+- `DirectEventHandler<T>` - For direct events (eg: `onClick`).
```

Comment:
Are these only Flow types? Do you know if these works in the same way for TypeScript or what are the equivalent TypeScript types?

Reply:
These types for typescript are defined here: https://github.com/DefinitelyTyped/DefinitelyTyped/pull/61466/files